### PR TITLE
Add fusonic/opengraph

### PR DESCRIPTION
**Pull request by juliushaertl**
Required for https://github.com/nextcloud/server/pull/33494

### CODE REVIEW

File: fusonic/linq/examples/1-average.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (nickvergessen) [inline fusonic/linq/examples/1-average.php:1]: should gitignore this folder and not commit the files inside

File: fusonic/linq/tests/bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (nickvergessen) [inline fusonic/linq/tests/bootstrap.php:1]: should gitignore this folder and not commit the files inside

File: fusonic/opengraph/examples/consume_website.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?php
```
Comment (nickvergessen) [inline fusonic/opengraph/examples/consume_website.php:1]: should gitignore this folder and not commit the files inside

File: fusonic/opengraph/phpunit.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<phpunit bootstrap="vendor/autoload.php">
```
Comment (nickvergessen) [inline fusonic/opengraph/phpunit.xml:1]: should gitignore this file and similar dev files and not commit the file

See the root .gitignore for inspiration which files should be excluded

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Review by come-nc — APPROVED** [review]

**Pull request merged**